images/linux/scripts/installers/docker-moby.sh

```diff
@@ -49,19 +49,28 @@ else
     set +e
 fi
 
-docker pull node:10
-docker pull node:12
-docker pull buildpack-deps:stretch
-docker pull buildpack-deps:buster
-docker pull node:10-alpine
-docker pull node:12-alpine
-docker pull debian:8
-docker pull debian:9
-docker pull alpine:3.7
-docker pull alpine:3.8
-docker pull alpine:3.9
-docker pull alpine:3.10
-docker pull ubuntu:14.04
+# Pull images
+images=(
```

Comment:
does it make sense to move docker images to toolset too?

Reply:
cc @maxim-lobanov

Reply:
I like @AlenaSviridenko's suggestion but I don't think that it should be in scope of this PR.
Could you please create work item on internal board to improve docker installation?